data/xml/2021.mrl.xml

```diff
@@ -275,8 +275,11 @@
       <author><first>Lucie</first><last>Flek</last></author>
       <pages>242–244</pages>
       <abstract>Interpolation-based regularisation methods have proven to be effective for various tasks and modalities. Mixup is a data augmentation method that generates virtual training samples from convex combinations of individual inputs and labels. We extend Mixup and propose DMix, distance-constrained interpolative Mixup for sentence classification leveraging the hyperbolic space. DMix achieves state-of-the-art results on sentence classification over existing data augmentation methods across datasets in four languages.</abstract>
-      <url hash="47db7ca9">2021.mrl-1.21</url>
+      <url hash="8996a24d">2021.mrl-1.21</url>
       <bibkey>sawhney-etal-2021-dmix</bibkey>
+      <revision id="1" href="2021.mrl-1.21v1" hash="47db7ca9"/>
+      <revision id="2" href="2021.mrl-1.21v2" hash="8996a24d" date="2021-11-16">Retracted.</revision>
+      <retracted date="2021-11-16">author wants to withdraw paper</retracted>
```

Comment:
This explanation should be more informative. What is the reason for the retraction?